### DISCUSSION TITLE
Jobs Menu display job as failed although it was successful

### DISCUSSION THREAD

**Discussion by abdullahodibat** [apache/airflow#20728] [Q&A]
Hello,
When i go to the `Browse` menu on the UI and then choose `Jobs`, i see some jobs as `failed`, although they are successful.
Any idea whats happening?
i am using airflow 2.0.1

Thanks in advance!

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!
